### PR TITLE
SM-5[API]: As a user, I want to have an endpoint to delete a sensor

### PR DESCRIPTION
**Pull request by dariocruz96**

### CODE REVIEW

File: API/templates/home.html [hunk 1]
```diff
@@ -14,11 +14,13 @@
        <tr>
         <th>Sensor Name</th>
         <th>Sensor Unit</th>
+        <th>Delete</th>
        </tr>
        {% for sensor in sensors %}
        <tr>
         <td>{{sensor[2]}}</td>
         <td>{{sensor[3]}}</td>
+        <td><a href="/delete/{{sensor[0]}}" class="btn btn-outline-danger btn-sm">Delete</a></a> </td>
```
Comment (tiagohorta21) [inline API/templates/home.html]: Why do we have a class "btn btn-outline-danger..." ? 
Why there are two closing </a> tags ?

File: API/main.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from django import db
```
Comment (tiagohorta21) [inline API/main.py]: Do we need imports from django ????????????

File: API/main.py [hunk 3]
```diff
@@ -54,5 +56,24 @@ def listSensor():
...
 
+@app.route('/delete/<id>')
+def deleteSensor(id):
+    # Open database connection
+    connection = sqlite3.connect("TP2.db")
+    cursor = connection.cursor()
+
+    cursor.execute("PRAGMA FOREIGN_KEYS = ON")
+    cursor.execute('DELETE FROM Sensor WHERE idSensor= ?', id)
+    # friend_to_delete = Sensor.query.get_or_404(id)
```
Comment (tiagohorta21) [inline API/main.py]: friend to delete ?

**Review by tiagohorta21 — CHANGES_REQUESTED** [review]

File: API/main.py (hunk 3 above)
Comment (dariocruz96) [reply API/main.py]: Opa

File: API/main.py (hunk 2 above)
Comment (dariocruz96) [reply API/main.py]: It was for the previous method used, changed to this new one so , not needed anymore. (Removed)

**Review by tiagohorta21 — APPROVED** [review]

**Pull request merged**